build.sbt

```diff
@@ -18,6 +18,7 @@ lazy val microservice = Project("catalogue-frontend", file("."))
     libraryDependencies ++= compile ++ test,
     // ***************
     RoutesKeys.routesImport ++= Seq(
+      "uk.gov.hmrc.cataloguefrontend.auth.AuthController",
```

Comment:
can we not just use the fully qualified controller in the routes? - we typically only use `routesImport` for model

Reply:
Removed